modules/odc-creating-functions.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-getting-started.adoc
+
+:_content-type: PROCEDURE
+[id="odc-creating-functions_{context}"]
+= Creating a function in the web console
+
+You can create a function from a Git repository by using the *Developer* perspective of the {product-title} web console.
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Serving are installed on the cluster.
+* The {pipelines-shortname} Operator is installed on the cluster.
+* You have logged in to the {product-title} web console.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have created or have access to a Git repository that contains the code for your function. The repostory must contain a `func.yaml` file and use the `s2i` build strategy.
+* Before you can create a function by using the web console, a cluster administrator must create the following tasks and pipeline, so that they are available for all namespaces on your cluster.
+** Create the `func-s2i` task:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/openshift-knative/kn-plugin-func/serverless-1.30/pkg/pipelines/resources/tekton/task/func-s2i/0.1/func-s2i.yaml
+----
+** Create the `func-deploy` task:
```

Comment:
Nitpick: I try to avoid imperative-mood procedure-step phrasing in the prerequisites (so that the module does not read like one procedure after another (in the prerequisites). For example, considering that you already have a couple of passive-voice points at the top of the prerequisites list, these three second-level list items could be reworked to something like this:
```suggestion
** The `func-deploy` task, which is created by running the following command:
```

Reply:
Passive voice should be avoided in the docs as much as possible - It's only used in those first two steps because those are things that can't be done by a developer, i.e. the Operator needs to already exist on the cluster before you do this procedure. I think it's better only to use the passive where it's absolutely necessary, and keep these commands as they are.

Reply:
Actually, maybe it makes sense to split the prerequisites into "cluster admin prereqs" vs developer prereqs for this procedure. Let me take a look at this again.